<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LGNE5KF" failures="1" tests="5" name="passengers-count-1" time="29.857" errors="0" timestamp="2024-02-07T02:34:26 IST">
  <testcase classname="com.atharvahiwase07.tests.flightreservation.FlightReservationTest" name="userRegistrationTest" time="1.859"/>
  <testcase classname="com.atharvahiwase07.tests.flightreservation.FlightReservationTest" name="registrationConfirmationTest" time="0.223"/>
  <testcase classname="com.atharvahiwase07.tests.flightreservation.FlightReservationTest" name="flightsSearchTest" time="0.43"/>
  <testcase classname="com.atharvahiwase07.tests.flightreservation.FlightReservationTest" name="flightSelectionTest" time="0.363"/>
  <testcase classname="com.atharvahiwase07.tests.flightreservation.FlightReservationTest" name="flightReservationConfirmationTest" time="0.161">
    <failure type="java.lang.AssertionError" message="expected [$500 USD] but found [$584 USD]">
      <![CDATA[java.lang.AssertionError: expected [$500 USD] but found [$584 USD]
at com.atharvahiwase07.tests.flightreservation.FlightReservationTest.flightReservationConfirmationTest(FlightReservationTest.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- flightReservationConfirmationTest -->
</testsuite> <!-- passengers-count-1 -->
